exporter/logicmonitorexporter/README.md

```diff
@@ -2,19 +2,18 @@
 
 | Status                   |                         |
 | ------                   | ------                  |
-| Stablilty                | traces [development](https://github.com/open-telemetry/opentelemetry-collector#development)  |
-|                          | logs [development](https://github.com/open-telemetry/opentelemetry-collector#development)                 |
-| Supported pipeline types | traces,logs             |
```

Comment:
Any reason for dropping support for traces?

Reply:
As support for traces in logicmonitor SDK is still under development, we decided to pull it off from this PR.
We will raise separate PR for support of traces once SDK gets developed.